src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Utils/ImporterUtils.cs

```diff
@@ -1,51 +1,64 @@
-﻿using System.Collections.Generic;
-using System.Linq;
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Data.Processor.Exceptions;
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Utils
 {
     public static class ImporterUtils
     {
-        public static readonly List<GeographicLevel> IgnoredGeographicLevels = new List<GeographicLevel>
+        /// <summary>
+        /// Levels which data will only be imported for if a file consists exclusively of data of that level.
+        /// </summary>
+        private static readonly List<GeographicLevel> SoloGeographicLevels = new()
         {
             GeographicLevel.Institution,
+            GeographicLevel.PlanningArea,
             GeographicLevel.Provider,
             GeographicLevel.School,
-            GeographicLevel.PlanningArea
         };
 
-        private static readonly List<GeographicLevel> AllowedSoloGeographicLevels = new List<GeographicLevel>
+        public static bool IsSoloGeographicLevel(this GeographicLevel geographicLevel)
```

Comment:
I'd be tempted to rename this. I feel like there is potential confusion with the new `IsSoleGeographicLevel` function. Not sure what though. `CanImportGeographicLevelSolo`, `IsImportableSoloGeographicLevel`, `IsSoleImportableOnlyGeographicLevel`?

Reply:
Have removed `ImporterUtils` now in the most recent commits and created separate `SoloImportableLevels`.
Methods of `ImporterUtils` that were getting values based on cols and rows have now joined other similar methods in `CsvUtil`.